api/ooniapi/aggregation.py

```diff
@@ -141,6 +142,10 @@ def get_aggregated() -> Response:
         in: query
         type: string
         description: Name of the tests, comma separated
+      - name: ooni_run_link_ids
```

Comment:
This should not include the `s` at the end. All other endpoints which deal with lists don't end with `s`

Reply:
Also it's inconsistent with the same parameter in measurements endpoint

Reply:
I renamed the variable in all API paths